app/components/ui/textarea.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+import { cn } from '~/lib/utils'
+
+export interface TextareaProps
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+
+const Textarea = React.forwardRef<HTMLTextAreaElement, TextareaProps>(
+  ({ className, ...props }, ref) => {
+    return (
+      <textarea
+        className={cn(
+          'flex h-60 w-full rounded-md border-none bg-white px-3 py-2 text-sm shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium file:text-foreground placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 resize-none',
+          className,
```

Comment:
**suggestion**: Consider allowing resizing of the textarea for better user experience, especially for longer text inputs.